From 0764e100a61033ca7a940c822e35e8c19dbc0944 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:27:28 -0400
Subject: [tools] perf/record: Remove advance_output function

Message-id: <1408462094-14194-280-git-send-email-jolsa@redhat.com>
Patchwork-id: 88218
O-Subject: [PATCH RHEL7.1 BZ1131394 279/325] perf record: Remove advance_output function
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit f34b9001f9a2f6fa41d3582fe515d194cc86bfb2
Author: David Ahern <dsahern@gmail.com>
Date: Wed Nov 6 11:41:35 2013 -0700

description
===========
1 line function with only 1 user; might as well embed directly.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/builtin-record.c b/tools/perf/builtin-record.c
index 2932069..19c4db6 100644
--- a/tools/perf/builtin-record.c
+++ b/tools/perf/builtin-record.c
@@ -77,11 +77,6 @@ struct perf_record {
  off_t   post_processing_offset;
 };
 
-static void advance_output(struct perf_record *rec, size_t size)
-{
- rec->bytes_written += size;
-}
-
 static int write_output(struct perf_record *rec, void *buf, size_t size)
 {
  struct perf_data_file *file = &rec->file;
@@ -461,7 +456,7 @@ static int __cmd_record(struct perf_record *rec, int argc, const char **argv)
     pr_err("Couldn't record tracing data.\n");
     goto out_delete_session;
    }
-   advance_output(rec, err);
+   rec->bytes_written += err;
   }
  }
 
-- 
1.7.1